inc/plc/class-did.php

```diff
@@ -167,17 +169,13 @@ protected function prepare_update_op() : ?SignedOperation {
 
 	/**
 	 * @throws Exception
-	 * @return Operation
+	 * @return Operation|WP_Error
 	 */
 	public function fetch_last_op() : Operation {
 		$url = sprintf( '%s/%s/log/last', static::DIRECTORY_API, $this->id );
-		$response = wp_remote_get( $url, [
-			'headers' => [
-				'Accept' => 'application/did+ld+json',
-			],
-		] );
+		$response = MiniFAIR\get_remote_url( $url );
 		if ( is_wp_error( $response ) ) {
-			throw new Exception( 'Error fetching last op: ' . $response->get_error_message() );
+			return $response;
```

Comment:
`prepare_update_op()` (~Line 131) will need to handle a `WP_Error` from this method